extensions/big-data-cluster/src/bigDataCluster/constants.ts

```diff
@@ -71,6 +71,7 @@ export namespace cssStyles {
 	export const selectedTabDiv = { 'border-bottom': '2px solid #000' };
 	export const unselectedTabDiv = { 'border-bottom': '1px solid #ccc' };
 	export const lastUpdatedText = { ...text, 'color': '#595959' };
+	export const errorText = { ...text, 'color': 'red' };
```

Comment:
Question: long term shouldn't these match theme values? How can we better enable this?

Reply:
I'm worried about this not looking right in dark theme, or other intermediate themes. Any way we can hook the theme values here?

Reply:
Yes, that's a known issue. I'm planning on addressing that when I go back and fix up the accessibility issues for this dialog since it doesn't handle theming well currently.